streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SparkAppBuildPipeServiceImpl.java

```diff
@@ -129,7 +129,7 @@ public class SparkAppBuildPipeServiceImpl
     private SparkApplicationInfoService applicationInfoService;
 
     @Autowired
-    private ApplicationLogService applicationLogService;
+    private SparkApplicationLogService applicationLogService;
 
     @Autowired
     private FlinkAppHttpWatcher flinkAppHttpWatcher;
```

Comment:
Whether you should use SparkAppHttpWatcher

Reply:
done